detectors/node/opentelemetry-resource-detector-alibaba-cloud/test/fixtures/use-alibaba-cloud-ecs-detector.js

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Is this file used anywhere?

Reply:
leftover from a previous approach on testing `suppressTracing`. no needed here